src/uwtools/drivers/global_equiv_resol.py

```diff
@@ -5,7 +5,7 @@
 from pathlib import Path
 from typing import List, Optional
 
-from iotaa import asset, external, task, tasks
+from iotaa import asset, external, tasks
```

Comment:
It's pretty cool that we've gotten some drivers all the way down to not needing their own task definitions.